content/zh-cn/docs/tutorials/services/source-ip.md

```diff
@@ -240,25 +227,12 @@ client_address=104.132.1.79
 
 * 客户端发送数据包到 `node2:nodePort`，它没有任何 endpoints
 * 数据包被丢弃
-* 客户端发送数据包到 `node1:nodePort`，它*有*endpoints
+* 客户端发送数据包到 `node1:nodePort`，它*有* endpoints
 * node1 使用正确的源 IP 地址将数据包路由到 endpoint
 
 
 用图表示：
```

Comment:
This whole file have no `<!-- original content -->`, could you add them back?
It will be huge in diff result, maybe we can split to another PR.

Reply:
Not sure about the priority.
@tengqm Do you think it should be done now or someday after the umbrella issue closed?

Reply:
Thanks for noticing. I retranslate this file and add original text